finagle-http/src/main/scala/com/twitter/finagle/http/filter/ModifyContentAndHeaderFieldFilter.scala

```diff
@@ -0,0 +1,55 @@
+package com.twitter.finagle.http.filter
+
+import com.twitter.finagle.http.{Fields, Request, Response, Status}
+import com.twitter.finagle._
+import com.twitter.util.Future
+
+/**
```

Comment:
could you add a test for this filter too?  end to end tests are rad, but for finding new bugs quickly, nothing beats a tightly scoped unit test =)